src/models/user.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Entity,
+  model,
+  property
+} from "@loopback/repository";
```

Comment:
use single quotes - please check this problem in all files modified by this patch